pandas/tests/extension/decimal/array.py

```diff
@@ -71,7 +71,15 @@ def dtype(self):
 
     @classmethod
     def _from_sequence(cls, scalars, dtype=None, copy=False):
-        return cls(scalars)
+        try:
+            return cls(scalars)
+        except TypeError:
+            # handling int 0 is necessary for groupby tests
```

Comment:
Can you explain why this is necessary? What is a code example running into this?

Reply:
this is made unnecessary by the fix i just pushed, so has been reverted